docker/rubrik-polaris-sdk-py3/build.conf

```diff
@@ -1 +1 @@
-version=1.0.0
+version=1.0.1
```

Comment:
There is no reason to bump this version as you did not really changed the docker. Please change it back and we can merge.